### ISSUE TITLE
About pre-trained model

### ISSUE THREAD

**Issue by re20100801** [Elody-07/AWR-Adaptive-Weighting-Regression#5]
Dear Author:
      Do you have an expect date to release your Pre-trained model? Looking forward for that a lot.
      I thought the pre-trained model is already there because you claim lots of detailed experiments ressult in your paper. But why it takes so long...   TT
      Hoping the released Model can reproduce your excellent performance!
      Thank you in advance.

**Comment by Elody-07** [comment]
Original pre-trained models are on the server in school but it is shut down due to 2019-nCoV. We are working on re-opening the machine.

**Comment by re20100801** [comment]
OK I see, Thank you anyway :D

**Comment by Elody-07** [comment]
We have uploaded the pretrained model of HourGlass.

**Comment by The-ArdaS** [comment]
Hi there,
When I try to test the pre-trained model, it says:

> loading model from ./results/hourglass_1.pth
> {'epoch': 14, 'MPE': 7.700112, 'AUC': 0.8504827899520097}

and the result is:

> [epoch -1][MPE 25.175][AUC 0.530]

I tried two times with Google Colab. One time I installed the requirements and the other time I did not. Both gave the same result. Did you use the last epoch(40)? Any help is appreciated.

(Btw, the hourglass_1 results (hourglass_1.txt) give the expected error value (7.7)).
